modules/querier/config.go

```diff
@@ -72,7 +72,7 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet)
 	cfg.Worker = worker.Config{
 		MatchMaxConcurrency:   true,
 		MaxConcurrentRequests: cfg.MaxConcurrentQueries,
-		Parallelism:           2,
+		Parallelism:           3,
```

Comment:
this won't have any impact b/c MatchMaxConcurrency is true. it might be hard to translate our cloud settings here

Reply:
reverted this change